src/__tests__/useLifecycleLogger.spec.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react';
+import { useLifecycleLogger } from '../hooks/useLifecycleLogger';
+
+describe('useLifecycleLogger', () => {
+  it('should be defined', () => {
+    expect(useLifecycleLogger).toBeDefined();
+  });
+});
+
```

Comment:
Let's also add test cases using  `@testing-library/react-hooks`. You can check our other hooks for examples @iampika . Great work so far!